### DISCUSSION TITLE
Discord community call: 6 January 2024

### DISCUSSION THREAD

**Discussion by BenJamesBen** [freesewing/freesewing#5532] [Support]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About

FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

### Details

Location: https://discord.freesewing.org/ (meeting room: 🔊voice chat)
Date: January 6th, 2024 5pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jan-6-2024/5pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- TBD

**Comment by darigovresearch** [comment]
If possible can we add to the agenda a discussion on ideas for new patterns ?

Since the new version is now live will be starting to try to make patterns again, most likely with something simple like an eye patch, see full list of old ideas below

https://github.com/freesewing/freesewing/discussions?discussions_q=label%3A%22%3Ashirt%3A+pattern+design%22+

**Comment by darigovresearch** [comment]
@tangerineshark do you need help/content for the quarterly email?

**Comment by darigovresearch** [comment]
Two new pattern design ideas that came from the call

- Chicken pattern weight - https://github.com/freesewing/freesewing/discussions/5775
- Tailor's ham/seam roll https://github.com/freesewing/freesewing/discussions/5774
